supplementary_style_guide/glossary_terms_conventions/general_conventions/h.adoc

```diff
@@ -149,7 +149,7 @@ Hidden replicas are primarily designed for services such as backups, bulk import
 [discrete]
 [[host-rhv]]
 ==== image:images/yes.png[yes] host (noun)
-*Description*: In Red Hat Virtualization, hosts are servers that provide the processing capabilities and memory resources used to run virtual machines. There are two types of hosts: Red Hat Enterprise Linux host and Red Hat Virtualization Host.
+*Description*: In Red Hat Virtualization, _hosts_ are servers that provide the processing capabilities and memory resources used to run virtual machines. There are two types of hosts: Red Hat Enterprise Linux host and Red Hat Virtualization Host.
```

Comment:
Is the difference in capitalization of the word 'host' intentional?

`Red Hat Enterprise Linux host and Red Hat Virtualization Host.`

Reply:
@ndeevy I'm not an expert in this area, but in taking a quick look at the RHEL and RHVH docs, it looks like this is correct.